### PR TITLE
Structural Changes

### PR DESCRIPTION
**Pull request by 06hpathak**

### CODE REVIEW

File: src/hooks/Content/useCreateNewContent.js
Comment (amitad16) [inline src/hooks/Content/useCreateNewContent.js:1]: are you deleting this one?

File: src/routes/MainRoutes.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ const MainRoutes = {
     },
     {
       path: 'activities',
-      element: <ProUserRoute />,
+      // element: <ProUserRoute />,
       children: [{ path: '', element: <Activities /> }]
```
Comment (amitad16) [inline src/routes/MainRoutes.js:94]: You can remove the children, and directly use <Activities /> in element

```suggestion
      element: <Activities />
```

File: src/pages/admin/Content/ContentPanel.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ import TableHead from '@mui/material/TableHead';
 import TableRow from '@mui/material/TableRow';
 
 import MuiAlert from '@mui/material/Alert';
-import useContentDeletion from 'hooks/Content/useContentDeletion';
+import useContentDeletion from 'hooks/Content/Admin Content/useContentDeletion';
```
Comment (amitad16) [inline src/pages/admin/Content/ContentPanel.js:32]: rename the `Admin Content` to `AdminContent`

File: src/pages/components-overview/Content.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { Grid } from '@mui/material';
 import ContentCard from './ContentCard';
-import useGetContent from '../../hooks/Content/useGetContent';
+
+import useGetContent from '../../hooks/Content/User Content/useGetContent';
```
Comment (amitad16) [inline src/pages/components-overview/Content.js:5]: Rename the `User Content` to `UserContent`

File: src/pages/admin/Activities/ActivitiesPanel.js [hunk 4]
```diff
@@ -19,7 +19,7 @@ import ModalComponent from '../Activities/CreateActivitiesModal'; // Import the
 import ViewActivitiesModal from './ViewActivitiesModal';
 import EditActivitiesModal from './EditActivitiesModal';
 import { PlusOutlined } from '@ant-design/icons';
-import usePaginationActivities from 'hooks/Activities/usePaginationActivities';
+import usePaginationActivities from 'hooks/Activities/Admin Activities/usePaginationActivities';
```
Comment (amitad16) [inline src/pages/admin/Activities/ActivitiesPanel.js:22]: Rename the `Admin Activities` to `AdminActivities`

File: src/hooks/Content/useCreateNewContent.js
Comment (06hpathak) [reply src/hooks/Content/useCreateNewContent.js:1]: no

File: src/routes/MainRoutes.js (hunk 1 above)
Comment (06hpathak) [reply src/routes/MainRoutes.js]: done

File: src/pages/admin/Content/ContentPanel.js (hunk 2 above)
Comment (06hpathak) [reply src/pages/admin/Content/ContentPanel.js]: done

File: src/pages/components-overview/Content.js (hunk 3 above)
Comment (06hpathak) [reply src/pages/components-overview/Content.js]: done

File: src/pages/admin/Activities/ActivitiesPanel.js (hunk 4 above)
Comment (06hpathak) [reply src/pages/admin/Activities/ActivitiesPanel.js]: done

**Review by amitad16 — APPROVED** [review]
LGTM

**Pull request merged**